lib/services/computeManagement/package.json

```diff
@@ -8,9 +8,12 @@
     "Georgiev, Yavor <yavorg@microsoft.com>",
     "Janczuk, Tomasz <tjanczuk@microsoft.com>",
     "Rodrigues, Andre <andrerod@microsoft.com>",
-    "Tavares, Chris <ctavares@microsoft.com>"
+    "Tavares, Chris <ctavares@microsoft.com>",
+    "Huang, Pengfei <phuang@microsoft.com>",
+    "Onalan, Alp <alpon@microsoft.com>",
+    "Lee, Hyonho <Hyonho.Lee@microsoft.com>"
   ],
-  "version": "0.10.1",
+  "version": "0.11.0",
```

Comment:
since we are here, could you please add a dev dependency of "underscrore" in this file   "underscore": "1.4.x"